packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -239,6 +233,137 @@ class OpenSettingsAction extends StatelessWidget {
   }
 }
 
+class DevToolsAboutDialog extends StatefulWidget {
+  @override
+  _DevToolsAboutDialogState createState() => _DevToolsAboutDialogState();
+}
+
+class _DevToolsAboutDialogState extends State<DevToolsAboutDialog> {
+  FlutterVersion _flutterVersion;
+  InfoController _controller;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _controller = InfoController(
+      onFlutterVersionChanged: (flutterVersion) {
+        if (!mounted) return;
+        setState(
+          () {
+            _flutterVersion = flutterVersion;
+          },
+        );
+      },
+    )..entering();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+
+    return AlertDialog(
+      actions: <Widget>[
+        FlatButton(
+          onPressed: () {
+            Navigator.of(context, rootNavigator: true).pop('dialog');
+          },
+          child: const Text('CLOSE'),
+        ),
+      ],
+      content: Column(
+        mainAxisSize: MainAxisSize.min,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Text('DevTools', style: textTheme.headline6),
+          const PaddedDivider(
+            padding: EdgeInsets.only(bottom: denseRowSpacing),
+          ),
+          const SelectableText('DevTools version ${devtools.version}'),
+          //
+          const SizedBox(height: defaultSpacing),
+          Text('Connected Device Info', style: textTheme.headline6),
```

Comment:
nit: what about 'Connected App Info'? Connected device makes me think it would give information like Android/iOS/Chrome/MacOs etc.